### PR TITLE
Create music_assistant.markdown

### PR DESCRIPTION
**Pull request by OzGav**
## Proposed change

This PR adds documentation to support the addition of the Music Assistant Integration into HA Core as per https://github.com/home-assistant/core/pull/128919

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/128919

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced documentation for integrating the Music Assistant with Home Assistant.
	- Detailed capabilities for automatic discovery of the Music Assistant Server and media player entity creation.
	- Explained how Home Assistant players can be duplicated in Music Assistant with options to hide originals.

- **Documentation**
	- Added comprehensive guidelines on using the Music Assistant integration, including available actions and options for media playback control.
	- Included instructions for manual configuration if automatic discovery fails.

### CODE REVIEW

File: source/_integrations/music_assistant.markdown [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+title: Music Assistant
+description: Instructions on how to integrate Music Assistant into Home Assistant.
+ha_category:
+  - Media player
+featured: true
+ha_release: 2024.11.0
```
Comment (joostlek) [inline source/_integrations/music_assistant.markdown]:
```suggestion
ha_release: 2024.11
```

File: source/_integrations/music_assistant.markdown [hunk 2]
```diff
@@ -0,0 +1,125 @@
+---
+title: Music Assistant
+description: Instructions on how to integrate Music Assistant into Home Assistant.
+ha_category:
+  - Media player
+featured: true
```
Comment (joostlek) [inline source/_integrations/music_assistant.markdown]:
```suggestion
```

**Comment by joostlek** [comment]
Please also make sure the CI succeeds, I think this is a problem with the order of the header

**Comment by OzGav** [comment]
I am sure this isn't the way to do this but I have made suggestions to fix all the CI and code rabbit issues but I can't commit my own.  Sorry I am new at this...

**Comment by joostlek** [comment]
Ah, it was caused by a wrong order of headings. I removed the actions for now since they are not included in what we have merged now, we will add that in a followup to keep the existing work flow

File: source/_integrations/music_assistant.markdown [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ha_codeowners:
+  - '@music_assistant'
+ha_domain: music_assistant
+ha_platforms:
+  - media_player
+ha_zeroconf: true
+ha_integration_type: integration
+---
+
+The Music Assistant (MA) integration allows you to connect Home Assistant to a [Music Assistant Server](https://music-assistant.io/). Once configured, all [MA Players](https://music-assistant.io/player-support/) show up as Home Assistant [media player entities](/integrations/media_player/).  Media players will allow you to control media playback and see the currently playing item.
```
Comment (c0ffeeca7) [inline source/_integrations/music_assistant.markdown]:
```suggestion
The **Music Assistant**(MA) {% term integrations %} allows you to connect Home Assistant to a [Music Assistant Server](https://music-assistant.io/). Once configured, all [MA Players](https://music-assistant.io/player-support/) show up as Home Assistant [media player entities](/integrations/media_player/).  Media players will allow you to control media playback and see the currently playing item.
```

File: source/_integrations/music_assistant.markdown [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+{% include integrations/config_flow.md %}
+
+### Manual configuration
+
+Under normal circumstances, Home Assistant will automatically discover your running Music Assistant Server. If there is something special about the HA or MA setup (e.g. the MA server is running as a remote Docker container) or discovery is not working, it is possible to manually specify the URL to your Music Assistant server. 
+
+## Media player
+
+The Music Assistant media player platform will create media player entities for all players available in MA including those imported from Home Assistant. This is needed to provide the full functionality Music Assistant has to offer. These entities will display media information, playback progress, and playback controls.
```
Comment (c0ffeeca7) [inline source/_integrations/music_assistant.markdown]:
```suggestion
The Music Assistant media player creates media player entities for all players available in MA including those imported from Home Assistant. This is needed to provide the full functionality Music Assistant has to offer. These entities will display media information, playback progress, and playback controls.
```

File: source/_integrations/music_assistant.markdown [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+## Notes
+
+- Any HA players added to MA will appear duplicated as the MA version of the player is created. The original HA player can be hidden if desired.
```
Comment (c0ffeeca7) [inline source/_integrations/music_assistant.markdown]:
```suggestion
- Any Home Assistant players added to Music Assistant will appear duplicated as the MA version of the player is created. The original HA player can be hidden if desired.
```

File: source/_integrations/music_assistant.markdown [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+- [Media player](#media-player)
+
+All of the Home Assistant [Media Player Control Actions](https://www.home-assistant.io/integrations/media_player/#media-control-actions) are supported.
+
+{% include integrations/config_flow.md %}
+
+### Manual configuration
+
+Under normal circumstances, Home Assistant will automatically discover your running Music Assistant Server. If there is something special about the HA or MA setup (e.g. the MA server is running as a remote Docker container) or discovery is not working, it is possible to manually specify the URL to your Music Assistant server.
```
Comment (c0ffeeca7) [inline source/_integrations/music_assistant.markdown]:
```suggestion
Under normal circumstances, Home Assistant automatically discovers your running Music Assistant Server. If something special about the HA or MA setup (for example, the MA server is running as a remote Docker container) or discovery is not working, you can manually specify the URL to your Music Assistant server. 
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @OzGav :+1:

**Pull request merged**